tests/components/test_proximity.py

```diff
@@ -607,8 +710,8 @@ def test_device_tracker_test1_nearest_after_test2_in_ignored_zone(self):
             'device_tracker.test2', 'work',
             {
                 'friendly_name': 'test2',
-                'latitude': 12.6,
-                'longitude': 7.6
+                'latitude': 100,
```

Comment:
I'll check your other comments tomorrow. This test needed a change because now the initial coordinates of both devices are changed.

> Op 11 feb. 2016 om 17:32 heeft Paulus Schoutsen <notifications@github.com> het volgende geschreven:
> 
> In tests/components/test_proximity.py:
> 
> > @@ -607,8 +710,8 @@ def test_device_tracker_test1_nearest_after_test2_in_ignored_zone(self):
> >              'device_tracker.test2', 'work',
> >              {
> >                  'friendly_name': 'test2',
> > -                'latitude': 12.6,
> > -                'longitude': 7.6
> > +                'latitude': 100,
> This is pretty much a big red flag. Why does this test needs updating, is it to work around a bug in the code?
> 
> —
> Reply to this email directly or view it on GitHub.
>